Sources_v3/StreamChatUI/Common Views/ChatSquareButton.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ChatSquareButton: Button {
+    // MARK: - Properties
+    
+    public var defaultIntrinsicContentSize: CGSize?
+    
+    public var image: UIImage? {
+        didSet {
+            updateContentIfNeeded()
+        }
+    }
```

Comment:
Where is this used? Why do we even have it when it doesn't allow setting different images for different control states?

Reply:
Yep, doesn't make sense, fixed.